Datamodel/constants.py

```diff
@@ -0,0 +1,8 @@
+from pathlib import Path
```

Comment:
Would be good to name this "datamodel" instead of "Datamodel"

https://peps.python.org/pep-0008/#package-and-module-names